src/Altinn.Platform/Altinn.Platform.Storage/Storage/Repository/DataRepository.cs

```diff
@@ -30,6 +32,10 @@ public class DataRepository : IDataRepository
         private readonly CloudBlobClient _blobClient;
         private readonly CloudBlobContainer _container;
 
+        private bool _useAppBlobClient = false;
+
+        private CloudBlobContainer AppBlobContainer { get; set; }
```

Comment:
Could be a field?

Reply:
Removed and moved to new datacontext object